beacon-chain/db/iface/interface.go

```diff
@@ -87,6 +88,7 @@ type NoHeadAccessDatabase interface {
 	SaveDepositContractAddress(ctx context.Context, addr common.Address) error
 	// SaveExecutionChainData operations.
 	SaveExecutionChainData(ctx context.Context, data *ethpb.ETH1ChainData) error
+	SaveSignedExecutionPayloadEnvelopeBlind(ctx context.Context, envelope *ethpb.SignedExecutionPayloadEnvelopeBlind) error
```

Comment:
Do we really need the Blind here? and even such a long name? any problem in calling this `SavePayloadEnvelope`?